package.json

```diff
@@ -384,6 +384,11 @@
             "PowerPlatform PCF Debugger for VS Code will use Microsoft Edge Dev version",
             "PowerPlatform PCF Debugger for VS Code will use Microsoft Edge Canary version"
```

Comment:
Can you please raise small PRs. I feel we could have done incremental changes. It helps in reviewing the PR also

Reply:
This one will be harder to de-tangle - BUT i am keeping corner cases coverage, telemetry and further test updates in a separate PR.